### ISSUE TITLE
Please Test: when I want to fill in a post session but my PIN hasn't been stored as part of the corresponding pre-session no questions are being displayed/ demographic part should only be displayed to users that haven't filled in this part yet

### ISSUE THREAD

**Issue by dupreesi** [yalla-coop/connect5#590]
- please check: 
- [ ] when filling in a post/follow up survey with a PIN that has not yet filled in the corresponding pre-survey -> user should get directed to correct survey (functionality is there but doesn't work currently)

- [ ] when filling in a survey that includes demographic questions with a PIN that has already filled in demographic details -> skip that section

**Comment by marwajomaa** [comment]
![Screenshot from 2019-09-25 13-28-11](https://user-images.githubusercontent.com/34250163/65593896-2ded6100-df9a-11e9-9f4b-d90112435da5.png)
@dupreesi can you explain the bug in video please ,

not sure what the bug is !  Is it when user wants to fill demographic question after he redirected to pre survey the next button is still disabled?! or anything else

**Comment by dupreesi** [comment]
Ok sorry, I've removed the label Bug. Its rather a check of 2 things (it didn't work for me when trying it before): 

1.) If a user uses a PIN for a survey that includes demographic data (usually pre-session-1 and pre-ttt) -> app checks if this PIN has previously filled in demographic data and if yes -> skips demographic data and jumps straight to the next section
bug here: When I used a PIN that has been stored the survey didn't load

2.) When a PIN is entered for a post session survey the app should check if there is a related pre-survey and if this PIN has already submitted this pre-survey. If not, the app asks the user to fill in the correct pre-survey before submitting the post-survey (this also didn't work for me) 
for example if a user wants to fill in post-survey-1 but has not been stored as a response of pre-survey-1 the app should notify the user with a popup and direct him/her to pre-survey-1 

does that make more sense? sorry for being unclear
